model/bin/w3_make

```diff
@@ -154,6 +155,7 @@
   cdf_programs="$cdf_programs ww3_ounp"
   cdf_programs="$cdf_programs ww3_bounc"
   cdf_programs="$cdf_programs ww3_trnc"
+  cdf_programs="$cdf_programs ww3_grid"
```

Comment:
Is there an option that says if ww3_grid should be a netcdf program or not?  The way this currently is, you are now requiring netcdf for ww3_grid, which essentially is now requiring netcdf as a library, no?   While I know there are ongoing discussions of integrating more netcdf, is there a way to make this optional or is it always essential now?

Reply:
@JessicaMeixner-NOAA It is required when SCRIPNC is in the switch. 
From a communication with @sbrus89: 
_The scrip.nc can be used with ESMF_regrid_gen_weights (or similar). I added this to aid in the creation of coupler mapping files for unstructured grids for E3SM. I wasn't sure if this was something that would be useful to include in the PR or not, so I decided to leave it in just in case. We can certainly leave it out if you think that would be better._

So, as Steven suggested, I can exclude SCRIP related developments, and we no longer need netcdf for ww3_grid, but I thought it might be helpful.

Reply:
Are all of the netcdf SCRIP related updates behind a switch?

Reply:
What I don't want to do, is require everyone even someone who is using all ascii files to now have a netcdf dependency. If we move all output to netcdf, then yes all programs will have netcdf dependency and it will be a requirement.  Even in the make_makefile script, this looks like an option, not a requirement, so when deciding if ww3_grid is a netcdf or regular program, shouldn't this also be an option?

Reply:
I realized it when I was checking mww3_test_04 with switch_PR3_UQ_MPI_SCRIPNC, so I had to modify the make_makefile.sh, the rest of tests went well without netcdf dependency, so I think these are behind SCRIPNC if I am not wrong.

Reply:
I think this needs to be moved into the if/else section starting on line  172: https://github.com/aliabdolali/WW3/blob/FB_Global_Unstr/model/bin/w3_make#L172

If it's scripnc then add ww3_grid to netcdf programs, if not, it should be a regular program.

Reply:
@JessicaMeixner-NOAA Thanks, it was a very helpful hint, I made the changes and ran the regtests again, all in good shape. Could you approve the review?